src/app/machines/views/MachineDetails/MachineConfiguration/MachineForm/MachineForm.tsx

```diff
@@ -109,6 +112,10 @@ const MachineForm = ({ systemId }: Props): React.ReactElement | null => {
             />
             <Definition description={machine.zone.name} label="Zone" />
             <Definition description={machine.pool.name} label="Resource pool" />
+            <Definition
+              children={<Icon name={machine.is_dpu ? "success" : "error"} />}
```

Comment:
I personally would use `success-grey` and `error-grey` here instead, but let's wait for UX on this

https://vanillaframework.io/docs/patterns/icons